src/OpenTelemetry.Instrumentation.ElasticsearchClient/Implementation/ElasticsearchRequestPipelineDiagnosticListener.cs

```diff
@@ -285,7 +283,7 @@ private string ParseAndFormatRequest(Activity activity, string debugInformation)
                     body = Encoding.UTF8.GetString(stream.ToArray());
                 }
 
-                return $"{method} {url}\r\n{body}";
```

Comment:
Does this change the expected contents of the tag `db.statement`? If yes, then that needs to be mentioned in the CHANGELOG as well.

Reply:
Updated